src/python/dxpy/scripts/dx.py

```diff
@@ -3787,6 +3787,117 @@ def publish(args):
     except:
         err_exit()
 
+def archive(args):
+    dx_archive_errors = [InvalidState, ResourceNotFound, PermissionDenied]
+
+    # resolve paths
+    target_project = None    
+    target_folder = None
+    target_files = []
+    for path in args.path:
+        try: 
+            project_id, folderpath, file_results = resolve_existing_path(path, expected_classes=None, allow_mult=True, all_mult=args.all)
+        except ResolutionError as e:
+            err_exit('Could not resolve "' + path + '": ' + e.msg, code=3, arg_parser=parser_archive)
```

Comment:
(Minor: we may want to move towards using other formatting methods / string interpolation a little more as it tends to be more robust to type issues and easier to deal with more complex formatting requirements)

Reply:
Sorry I was lazy copypasting the format from other resolving error messages. I will change that to .format string.